lte/gateway/c/core/oai/tasks/amf/deregistration_request.cpp

```diff
@@ -265,7 +271,10 @@ void amf_remove_ue_context(ue_m5gmm_context_s* ue_context_p) {
 
   if (found_ue_id != ue_context_map.end()) {
     OAILOG_DEBUG(
-        LOG_AMF_APP, "Removed ue id = %u entry from the ue context map\n",
+        LOG_AMF_APP,
+        "Removed ue id = " AMF_UE_NGAP_ID_FMT
+        " entry from the "
+        "ue context map\n",
         ue_context_p->amf_ue_ngap_id);
```

Comment:
looks too many lines, is this formatted by linter?

Reply:
Comment addressed, I think it was auto formatted.